src/eval.ml

```diff
@@ -92,6 +92,10 @@ and whnf_stk : term -> stack -> term * stack = fun t stk ->
   | (Abst(_,f), u::stk) ->
       Pervasives.incr steps;
       whnf_stk (Bindlib.subst f u) stk
+  (* Let unfolding *)
+  | (LLet(t,_,u), stk ) ->
+      Pervasives.incr steps;
```

Comment:
don't we need to check that u is not closed?

Reply:
Perhaps we could raise a warning while scoping?

Reply:
Good idea.